modules/structs/pull.go

```diff
@@ -95,3 +95,14 @@ type EditPullRequestOption struct {
 	RemoveDeadline      *bool      `json:"unset_due_date"`
 	AllowMaintainerEdit *bool      `json:"allow_maintainer_edit"`
 }
+
+// ChangedFile store information about files affected by the pull request
+type ChangedFile struct {
+	NameHash    string `json:"name_hash"`
+	Filename    string `json:"filename"`
+	Status      string `json:"status"`
+	Additions   int    `json:"additions"`
+	Deletions   int    `json:"deletions"`
+	HTMLURL     string `json:"html_url,omitempty"`
```

Comment:
IMO `HTMLURL` is a pretty bad name (unclear, although it was used in history). I am not sure whether people really like this name .....

As a new API, could there be a better name? Just a question, not a blocker.